src/axom/mir/MIRAlgorithm.cpp

```diff
@@ -24,14 +24,7 @@ void MIRAlgorithm::execute(const conduit::Node &n_input,
   const auto domains = conduit::blueprint::mesh::domains(n_input);
   if(domains.size() > 1)
   {
-    // Handle multiple domains
-    for(const auto &dom_ptr : domains)
-    {
-      const conduit::Node &n_domain = *dom_ptr;
-      conduit::Node &n_newDomain = n_output.append();
-
-      executeSetup(n_domain, n_options, n_newDomain);
-    }
+    SLIC_ERROR("The input node contains multiple domains. Pass a single domain at a time instead.");
```

Comment:
Do we have a multidomain example that loops over its domains and calls MIR on each one?

Reply:
I'm changing one of the examples to do that now. It might spark some other changes.